### DISCUSSION TITLE
List methods returning a stream with one only item

### DISCUSSION THREAD

**Discussion by gabrieloliveirabrito** [SpinlockLabs/github.dart#319] [General]
I've been used this package to list the issues of an repository, bug another package wants the Stream that read a list of objects, but this package return a Stream with only one object. To read all data, i need to convert the stream to future, or i receiving really only one item.

**Comment by ghost** [comment]
> I've been used this package to list the issues of an repository, bug another package wants the Stream that read a list of objects, but this package return a Stream with only one object. To read all data, i need to convert the stream to future, or i receiving really only one item.
